Operational-Procedures/Scheduler/night-time-operations/index.rst

```diff
@@ -0,0 +1,255 @@
+.. _scheduler-night-time-operation:
+
+##############################
+Scheduler Night-time Operation
+##############################
+
+.. _scheduler-night-time-operation-initial-optical-alignement:
+
+Initial Optical Alignement
+==========================
+
+When the Scheduler is initialized at the beginning of the night, the first thing it will perform is an optical alignement, by running a curvature wavefront sensing script.
+This is a special configuration done on the *scheduling algorithm* side alonside the survey operation.
```

Comment:
is configuration the right word here? I'm worried it'll be confused with the "configurationApplied" parameters etc.

Reply:
I would use the word "operation". Or you can use the word "action."